app/controllers/subscriptions_controller.rb

```diff
@@ -0,0 +1,98 @@
+class SubscriptionsController < ApplicationController
+  before_action :check_logon
+
+  before_action :set_subscription, only: %i[ show edit update destroy ]
+  
+  before_action :set_forum, only: %w[new create]
+
+  # GET /subscriptions or /subscriptions.json
+  def index
+    @subscriptions = Subscription.where(user_id: @current_user.id).includes(:forum)
```

Comment:
You don't think you need this line since subscriptions#index by design returns a collection of forum objects. (If you're worried about the `@subscriptions` reference in the jbuilder file, I'd rather change that to `@forums` (defined below) so that you're not returning completely different data depending on whether you're using a browser or an API client.)